i-slide.js

```diff
@@ -7,8 +7,11 @@ if (params.get("selector")) {
   document.querySelectorAll(params.get("selector")).forEach(
     el => {
       const newEl = document.createElement("i-slide");
-      const src = el.getAttribute("href") || el.getAttribute("src") || el.dataset["islideSrc"];
+      const src = el.getAttribute("href") || el.getAttribute("src") || el.dataset["islidesrc"];
```

Comment:
I don't think that's correct. The capitalization of `islideSrc` comes from HTML processing rules of `data-` attributes: a `data-islide-src` attribute can be read through `.dataset["islideSrc"]`. We would capture `data-islidesrc` otherwise, and that wouldn't match the README.